app/kumactl/pkg/output/json/printer.go

```diff
@@ -16,7 +16,7 @@ var _ output.Printer = &printer{}
 type printer struct{}
 
 func (p *printer) Print(obj interface{}, out io.Writer) error {
-	b, err := json.Marshal(obj)
+	b, err := json.MarshalIndent(obj, "", "  ")
 	if err != nil {
 		return err
 	}
```

Comment:
Let's add a newline to the end (for consistency with `YAML` and `TABLE` output)

Reply:
What do you mean by add a newline to the end ?
Tell me if I'm wrong, you want a newline to the end of the prettyprint output ? Am I right ?

Reply:
Yes, JSON should be followed by a newline.

E.g., 

```
b, err := json.MarshalIndent(obj, "", "  ")
if err != nil {
	return err
}
if _, err := out.Write(b); err != nil {
	return err
}
_, err = fmt.Fprintln(out)
return err
```